packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -641,7 +641,7 @@ abstract class ClusterBase extends Resource implements ICluster {
    * @param manifest a list of Kubernetes resource specifications
    * @returns a `KubernetesResource` object.
    */
-  public addManifest(id: string, ...manifest: any[]): KubernetesManifest {
+  public addManifest(id: string, ...manifest: Record<string, any>[]): KubernetesManifest {
```

Comment:
The type should also be changed here:

https://github.com/aws/aws-cdk/blob/2e067d7d00b10b4c5e26665386c0a86340a83379/packages/%40aws-cdk/aws-eks/lib/k8s-manifest.ts#L42